client/src/EventsList.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { PollingEvent, EventsListProps } from "./types.ts";
```

Comment:
kdyz to spustim v prohlizeci dostanu jen
```
Uncaught SyntaxError: The requested module '/src/types.ts' does not provide an export named 'EventsListProps' (at EventsList.tsx:3:24)
```

je treba opravit ty importy

import { type PollingEvent } from "./types";

v App.tsx

a pak 

import { type EventsListProps } from "./types.ts";

v EventsList